functions/AutoRemediateOrchestrator.js

```diff
@@ -15,40 +14,35 @@ module.exports.handler = (event, context, callback) => {
       'The %s is not supported. Exiting gracefully ...',
       AutoRemediate
     )
-    callback(null)
-    return
+    return null
   }
   if (!CONFIG[`${AutoRemediate}`]['enabled']) {
     console.log('The %s is not enabled. Exiting gracefully ...', AutoRemediate)
-    callback(null)
-    return
+    return null
```

Comment:
♻️ Same as comment above
```suggestion
    return;
```